src/Esfa.Vacancy.Register.UnitTests/SearchApprenticeship/Api/Mappings/GivenASearchApprenticeshipParametersMapper/WhenMappingDistanceInMiles.cs

```diff
@@ -15,7 +15,7 @@ public class WhenMappingDistanceInMiles
         [SetUp]
         public void Setup()
         {
-            var config = AutoMapperConfig.Configure();
+            MapperConfiguration config = AutoMapperConfig.Configure();
```

Comment:
😢 I think vars are better here. Best to get a team decision on this else we'll be changing back and forth.

Reply:
Reverted changes